podcasts/WatchManager+Send.swift

```diff
@@ -23,14 +23,17 @@ extension WatchManager {
             return
         }
 
+        // Hold a local reference so we don't potentially run into a deallocated `self` when the below blocks are run.
+        let cachedLog = self.cachedLog
+
         // since we don't know how long it takes for a send message to timeout, wait only 10 seconds for a watch response before giving up here
         var haveCalledCompletion = false
-        logFileRequestTimedAction.startTimer(for: 5.seconds) { [weak self] in
+        logFileRequestTimedAction.startTimer(for: 5.seconds) { [cachedLog] in
```

Comment:
Keep a local copy in case self has been deallocated. It shouldn't be changing in the interim anyway and its a cached copy so isn't expected to be the most up to date.